### ISSUE TITLE
[ReviOS Tool v1.12.1 and up] "Unsupported build detected" error on Windows 11 Pro 24H2

### ISSUE THREAD

**Issue by syrnyk** [meetrevision/revision-tool#96]
**Describe the bug**
After installing the Revision Tool on Windows 11 Pro 24H2, I have an the "Unsupported build detected" error upon opening the Revision Tool. I don't know if it's relevant, but I was previously running Windows 11 Pro 23H2 with ReviOS 11 23.10. I then downloaded the Windows ISO for 24H2 (retrieved from the ReviOS update instruction page), followed the update instructions on ReviOS website (https://revi.cc/docs/playbook/upgrade/). When 24H2 was installed, I downloaded the latest ReviOS tool in order to run it and disable Windows Defender, and finally, got this error when trying to run v1.12.1, v1.99.9 and v1.99.10.

**Expected behavior**
I do not know if 24H2 is supposed to be supported yet. Maybe it's a bug or it simply needs 24H2 support to be implemented.

**Screenshots**
![image](https://github.com/user-attachments/assets/6e00b46c-f7e4-49c8-9004-9030f040a7e3)
![image](https://github.com/user-attachments/assets/53d8d545-f729-44aa-a6e9-dbb74a6747e6)

**Desktop (please complete the following information):**
 - OS: Windows 11 Pro 24H2 26100.1742 (Feature pack: 1000.26100.18.0)
 - Revision Tool Version: 1.12.1 and up

Thanks !

**Comment by Edgars-Cirulis** [comment]
Hello there, based on your description, it seems you haven't applied the [Revi Playbook](https://revi.cc/docs/playbook/general). The tool requires the Revi Playbook to be applied on Windows. It does not support standalone usage.

**Comment by syrnyk** [comment]
Thanks for the feedback. It's the first time I perform the update process and the guide said to update Windows, run the tool to disable Windows Defender, then apply the playbook. 

I thought the "apply tool" part was necessary to re-run the playbook on the updated OS. At least that's how I interpreted it, but the guide probably assumes that the old ReviOS playbook is still installed after Windows upgrade, which was not the case for me when going from 23H2 to 24H2.

My bad, it's just a step order problem on my part then. 
I applied the playbook 24.06, installed the tool and everything works as expected.
